bety_brapi/controllers_impl/GermplasmController_impl.py

```diff
@@ -39,3 +39,84 @@ def germplasm_search_get(germplasmPUI, germplasmDbId, germplasmName, commonCropN
 
     # return the resulting data
     return helper.create_result({"data": data}, count, pageSize, page)
+
+def treatments_by_experiment_get(experimentId):
+
+    params = list()
+
+    # get all sitegroups and sites
+    query = ""
+
+    query = "SELECT experiments_treatments.experiment_id as experimentId, " \
+            "   experiments_treatments.treatment_id as treatmentId, " \
+            "   treatments.name as treatmentName, " \
+            "   treatments.definition as treatmentDefinition " \
+            "FROM experiments_treatments INNER JOIN treatments ON experiments_treatments.treatment_id=treatments.id " \
+            "WHERE experiments_treatments.experiment_id = "+experimentId
+
+    print(query)
```

Comment:
remove debug code here and other places where you have a print statement.

Reply:
Sure thing. I just committed the cultivars part, will clean up everything now.